docs/specs/xref.yml

```diff
@@ -589,6 +589,34 @@ outputs:
     {"message_severity":"warning","code":"duplicate-uid","message":"UID 'b' is duplicated in 'docs/a.json(6,14)', 'docs/b.json(3,12)'.","file":"docs/a.json","line":6,"column":14}
     {"message_severity":"warning","code":"duplicate-uid","message":"UID 'b' is duplicated in 'docs/a.json(6,14)', 'docs/b.json(3,12)'.","file":"docs/b.json","line":3,"column":12}
 ---
+# uid conflict globally
+inputs:
+  docfx.yml: |
+    xref:
+    - 1.xrefmap.json
+  1.xrefmap.json: |
+    {
+      "repositoryUrl": "a_repo",
+      "references":[{
+          "uid": "a"
+      }]
+    }
+  docs/a.json: |
+    {
+      "$schema": "https://raw.githubusercontent.com/dotnet/docfx/v3/schemas/TestData.json",
+      "uid": "a"
+    }
+  _themes/ContentTemplate/schemas/TestData.schema.json: |
+    {
+      "properties": {
+        "uid": { "contentType": "uid", "UIDGlobalUnique": true }
```

Comment:
shall we use enum value like `docset`, `global` here?

Reply:
Yufei have suggested to change the enum type to bool. @yufeih